### PR TITLE
Added and fixed tests for producers platform, small fixes

### PR DESCRIPTION
**Pull request by stephanegigandet**

### CODE REVIEW

File: cgi/export_products.pl [hunk 1]
```diff
@@ -70,6 +70,10 @@
 	$html .= "<p>" . lang("producers_platform_licence") . "</p>";
 	$html .= "<p>" . lang("export_product_data_photos_please_check") . "</p>";
 
+	# Display button for admins only
+
+	if ($User_id =~ /-off$/) {
```
Comment (VaiTon) [inline cgi/export_products.pl]: Why not use `isAdmin()`?

File: lib/ProductOpener/Producers.pm [hunk 2]
```diff
@@ -413,6 +415,22 @@ sub init_fields_columns_names_for_lang($) {
...
+	"g" => "g",
+	"gr" => "g",
+	"grams" => "g",
+	"grammes" => "g",
+	"mg" => "mg",
+	"mcg" => "mcg",
+	"percent" => "percent",
+	"kj" => "kj",
+	"kcal" => "kcal",
+	"cal" => "kcal",
```
Comment (VaiTon) [inline lib/ProductOpener/Producers.pm]: cal is 1/1000 of a kcal

Comment (stephanegigandet) [reply lib/ProductOpener/Producers.pm]: This is not a conversion table, it is a list of synonyms used by producers when they transmit us data. In practice absolutely no producer uses cal (as in 1/1000 of kcal) as a unit for energy. When they have "cal" or "calories" in the header of a column, they always mean kcal. The units in this table are lowercased, so it's more for the "big Calories".

File: cgi/export_products.pl (hunk 1 above)
Comment (stephanegigandet) [reply cgi/export_products.pl]: Good point. I'll make things cleaner, I need to implement https://docs.google.com/document/d/154MN36FACbS3PqSaEcsoerSz0n6jo9RwaD3eLRChNgI/edit first though.

Comment (stephanegigandet) [reply cgi/export_products.pl]: Good point, I'll make things cleaner. I need to implement https://github.com/openfoodfacts/openfoodfacts-server/issues/2085 first.

File: icons/arrow_upward.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+material/baseline-arrow_upward-24px.svg
```
Comment (VaiTon) [inline icons/arrow_upward.svg]: Why don't we just rename the file?

File: lib/ProductOpener/Producers.pm (hunk 2 above)
Comment (VaiTon) [reply lib/ProductOpener/Producers.pm]: Ok, just to make it obvious we could add a comment explaining what you wrote here.

File: icons/arrow_upward.svg (hunk 3 above)
Comment (stephanegigandet) [reply icons/arrow_upward.svg]: To keep track that it's a material icon.

File: lib/ProductOpener/Producers.pm (hunk 2 above)
Comment (stephanegigandet) [reply lib/ProductOpener/Producers.pm]: Good idea, I'll add it.

**Pull request merged**